### PR TITLE
Update review section of design principles

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by Hixie** [comment]
cc @tvolkert

File: design-principles.md [hunk 1]
```diff
@@ -49,9 +49,14 @@ also many other benefits of code reviews:
...
 
+We recommend you consider
+[these suggestions](https://testing.googleblog.com/2017/06/code-health-too-many-comments-on-your.html)
+for dealing with code reviews.
+ 
 If you're working on a big patch, don't hesitate to get reviews early,
 before you're ready to check code in. Also, don't hesitate to ask for
-multiple people to review your code.
+multiple people to review your code, and don't hesitate to provide unsolicited
+comments on other people's PRs. The more review the better.
```
Comment (tvolkert) [inline design-principles.md]: reviews

**Comment by Hixie** [comment]
I don't really understand how to make `continuous-integration/travis-ci/push` pass. It's testing against an old state of the website.

**Comment by Hixie** [comment]
Gonna land as-is and hope that I am right that this won't break the build...

**Pull request merged**